jrcc-access-spring-boot-autoconfigure/src/main/java/ca/bc/gov/open/jrccaccess/autoconfigure/plugins/console/ConsoleOutput.java

```diff
@@ -34,7 +34,7 @@ public ConsoleOutput(Prettifier prettifier) {
 	public void send(String content, TransactionInfo transactionInfo) {
 		logger.info("transactionInfo: {}", transactionInfo);
 		String result = this.prettifier.prettify(content);
-		System.out.println(result);
```

Comment:
actually want to keep this one this way, it's just the output, technically not a log...

Reply:
Okay kinda guessed that would be the case - just wanted to check 😄